stellar-multitenant/pkg/internal/httphandler/tenants_handler.go

```diff
@@ -44,3 +49,49 @@ func (t TenantsHandler) GetByIDOrName(w http.ResponseWriter, r *http.Request) {
 
 	httpjson.RenderStatus(w, http.StatusOK, tnt, httpjson.JSON)
 }
+
+func (h TenantsHandler) PostTenants(rw http.ResponseWriter, req *http.Request) {
+	ctx := req.Context()
+
+	var reqBody *validators.TenantRequest
+	if err := httpdecode.DecodeJSON(req, &reqBody); err != nil {
+		log.Ctx(ctx).Errorf("decoding request body: %v", err)
+		httperror.BadRequest("", err, nil).Render(rw)
+		return
+	}
+
+	validator := validators.NewTenantValidator()
+	reqBody = validator.ValidateCreateTenantRequest(reqBody)
+	if validator.HasErrors() {
+		httperror.BadRequest("invalid request body", nil, validator.Errors).Render(rw)
+		return
+	}
+
+	tnt, err := h.ProvisioningManager.ProvisionNewTenant(
+		ctx, reqBody.Name, reqBody.OwnerFirstName,
+		reqBody.OwnerLastName, reqBody.OwnerEmail, reqBody.OrganizationName,
+		reqBody.SDPUIBaseURL, reqBody.NetworkType,
```

Comment:
That's one thing we need to change. We've had a feedback when presenting this design that `NetworkType` should be an instance-level config (same as single tenant) instead of a tenant-level config.

So a host will have to deploy an SDP testnet instance and an SDP pubnet instance if they want tenants on both.

Reply:
Got it! Here comes the `NetworkPassphrase` thing! Thank you so much for the feedback.

Reply:
Changed: [82adf47](https://github.com/stellar/stellar-disbursement-platform-backend/pull/97/commits/82adf47a5a0c73a4f4f61773d3cbc3cf98cf9d59)